raiden-ts/src/services/actions.ts

```diff
@@ -45,3 +45,30 @@ export interface iouClear extends ActionType<typeof iouClear> {}
 
 export const udcDeposited = createAction('udc/deposited', UInt(32));
 export interface udcDeposited extends ActionType<typeof udcDeposited> {}
+
+const UdcWithdrawId = t.type({
+  amount: UInt(32),
+});
+
+export const udcWithdraw = createAsyncAction(
+  UdcWithdrawId,
+  'udc/withdraw/request',
+  'udc/withdraw/success',
+  'udc/withdraw/failure',
+  t.type({}),
```

Comment:
Nitpick: when an action doesn't accept a payload, I usually use `t.undefined` instead of an empty type